examples/reproduce_testcases.jl

```diff
@@ -13,30 +13,30 @@ Langevin dynamics.
 """
 function test_diamond_heisenberg_sf()
     crystal = Sunny.diamond_crystal()
-    J = 28.28           # Units of K
+    J = 7.5413   # Units of K
     interactions = [
-        heisenberg(J, Bond(3, 6, [0,0,0])),
+        heisenberg(J, Bond(1, 3, [0,0,0])),
     ]
-    sys = SpinSystem(crystal, interactions, (8, 8, 8))
+    sys = SpinSystem(crystal, interactions, (8, 8, 8), [SiteInfo(1, 3/2, I(3))])
     rand!(sys)
 
-    Δt = 0.02 / J       # Units of 1/K
-    kT = 4.             # Units of K
+    Δt = 0.02 / ((3/2)^2 * J) # Units of 1/K
```

Comment:
The motivation for writing it like this was because the example documentation I write:

"A rough rule of hand is that a reasonable value for integration timestep size is Δt≈0.02/(S^2∗J)"

and I just wanted to be explicit about how the Δt was being obtained.

Reply:
Maybe do it in two steps then?
```
S = 3/2
...
Δt = 0.02 / (S^2 * J)
```